
{'date': 'Fri Jun 29 2018 11:42:34.902 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 2
    Alexandre
NameError: name 'Alexandre' is not defined
'''},
{'date': 'Fri Jun 29 2018 11:47:16.417 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 2
    Alexandre
NameError: name 'Alexandre' is not defined
'''},
{'date': 'Fri Jul 06 2018 10:32:55.942 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 11
    historia()
  module <module> line 9
    pessoa.entra(Cenapraia)
NameError: name 'Cenapraia' is not defined
'''},
{'date': 'Fri Jul 06 2018 10:54:02.344 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module <string> line 9
  txtpessoa = Texto(cenapraia, "bom dia”)
                                         ^
SyntaxError: EOL while scanning string literal
'''},
{'date': 'Fri Jul 06 2018 10:56:27.161 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module <string> line 9
  txtpessoa = Texto(cenapraia, "bom dia”)
                                         ^
SyntaxError: EOL while scanning string literal
'''},
{'date': 'Fri Jul 06 2018 10:59:07.48 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module <string> line 9
  txtpessoa = Texto(cenapraia, "Bom Dia!”)
                                          ^
SyntaxError: EOL while scanning string literal
'''},
{'date': 'Fri Jul 06 2018 11:00:09.120 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module <string> line 9
  txtpessoa = Texto(cenapraia, "Bom Dia”)
                                         ^
SyntaxError: EOL while scanning string literal
'''},
{'date': 'Fri Jul 06 2018 11:01:07.278 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module <string> line 9
  txtpessoa = Texto(cenapraia,"Bom Dia”)
                                        ^
SyntaxError: EOL while scanning string literal
'''},